src/main/java/org/Olivia/EventDispatcher.java

```diff
@@ -0,0 +1,221 @@
+                this.table.deleteEntry(Integer.parseInt(args[1]))+"\n"));
+        disk.syncToFile(this.table);
+        return 200;
+    }
+
+    private int addEntryToCalendar(String input) throws Exception {
+        String[] args=input.toLowerCase().split(" ");
+        if (args[0].equals("todo") && args.length>=2){
+            CalendarEntryTodo entry=new CalendarEntryTodo(input.substring(5));
+            int status=this.table.addEntry(entry);
+            if (status==200){
+                ui.cout(UiHandler.generateSection("Verstehe, added: "+entry.toString()+"\n"));
+            }
+            disk.syncToFile(this.table);
+            return status;
+        }
+        else if (args[0].equals("deadline") && args.length>=4){
+            if (input.indexOf("/by")==-1){
+                throw new InvalidParameterException("Sorry, what is the exact time of the deadline?\nCheck the help message for information on command syntax");
+                //return 500;
+            }
+            input=input.substring(9);
+            String time=input.substring(input.indexOf("/by")+4);
+            String title=input.substring(0, input.indexOf("/by")-1);
+            CalendarEntryDeadline entry=new CalendarEntryDeadline(title, time);
+            int status=this.table.addEntry(entry);
+            if (status==200){
+                ui.cout(UiHandler.generateSection("Verstehe, added: "+entry.toString()+"\n"));
+            }
+            disk.syncToFile(this.table);
+            return status;
+        }
+        else if (args[0].equals("event") && args.length>=4){
+            if (input.indexOf("/at")==-1 || input.indexOf(" - ")==-1){
+                throw new InvalidParameterException("Sorry, what is the exact time of the event?\nCheck the help message for information on command syntax");
+                //return 500;
+            }
+            input=input.substring(6);
+            String time=input.substring(input.indexOf("/at")+4);
+            String title=input.substring(0, input.indexOf("/at")-1);
+            CalendarEntryEvent entry=new CalendarEntryEvent(title, time.split(" - ")[0], time.split(" - ")[1]);
+            int status=this.table.addEntry(entry);
+            if (status==200){
+                ui.cout(UiHandler.generateSection("Verstehe, added: "+entry.toString()+"\n"));
+            }
+            disk.syncToFile(this.table);
+            return status;
+        }
+        throw new InvalidParameterException("Sorry, I don't seem to understand you");
+        //return 500;
+    }
+
+    private int find(String input){
+        input=input.substring(5);
+        List<CalendarEntry> entries = this.table.getEntriesContains(input);
+        String content="";
+        for (CalendarEntry e: entries){
+            content=content+e.toString()+"\n";
+        }
+        ui.cout((UiHandler.generateSection(content)));
+        return 200;
+    }
+
+    /**
+     * calls corresponding functions based on the input command
```

Comment:
I think these documentation should follow the recommended Java style guide (https://se-education.org/guides/conventions/java/intermediate.html), e.g. calls should be capitalized, and sentence should end with a fullstop.